new-getting-started-guides.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'New Getting Started Guides: Astro, Remix, SolidStart, SvelteKit, Next.js, and Nuxt'
+description: 'New getting started guides are now available for Astro, Remix, SolidStart, SvelteKit, Next.js, and Nuxt.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/launch-week.jpg
+  alt: 'Launch Week'
+author: Alex Francoeur
+date: 09-01-2023
+published: true
+slug: new-get-started-with-xata-guides
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/launch-week-og.jpg
+---
+
+Our mission at Xata is to radically simplify the way developers work with data. This is ingrained in everything we do, from
```

Comment:
Removed "radically" since it's a bit of a startup cringe. I know we use it on the about page, but that can be cringey :)

```suggestion
Our mission at Xata is to simplify the way developers work with data. This is ingrained in everything we do, from
```